src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
+
+
+    with self.argument_context('sb namespace create') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
+        c.argument('tags', options_list=['--tags', '-t'], arg_type=tags_type, help='tags for the namespace in '
+                                                                                   'Key value pair format')
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(['Basic', 'Standard', 'Premium']),
+                   help='Sku name and allowed values are, Basic, Standard, Premium')
+        c.argument('location', options_list=['--location', '-l'], help='Location')
+        c.argument('skutier', option_list=['--skutier'], arg_type=get_enum_type(['Basic', 'Standard', 'Premium']),
+                   help='Sku Tire with allowed values are, Basic, Standard, Premium')
+        c.argument('capacity', option_list=['--capacity'], help='Capacity for Sku')
+
+
+    # region Namespace Get
+    for scope in ['sb namespace get', 'sb namespace delete']:
+        with self.argument_context(scope) as c:
+            c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+            c.argument('namespace_name', options_list=['--name', '-n'], required=True, help='name of the Namespace')
+
+
+    with self.argument_context('sb namespace list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=False)
+
+
+    # region Namespace Authorizationrule
+    for scope in ['sb namespace authorizationrule create',
+                  'sb namespace authorizationrule get',
+                  'sb namespace authorizationrule listkeys',
+                  'sb namespace authorizationrule regeneratekeys',
+                  'sb namespace authorizationrule delete']:
+        with self.argument_context(scope) as c:
+            c.argument('resource_group_name', arg_type=resource_group_name_type, required=True),
+            c.argument('namespace_name', options_list=['--namespace-name'], required=True,
+                       help='name of the Namespace'),
+            c.argument('authorization_rule_name', options_list=['--name'], required=True,
+                       help='name of the Namespace AuthorizationRule')
+
+
+    with self.argument_context('sb namespace authorizationrule create') as c:
+        c.argument('accessrights', options_list=['--access-rights'], required=False,
+                   help='Authorization rule rights of type list, allowed values are Send, Listen or Manage')
+
+    with self.argument_context('sb namespace authorizationrule regeneratekeys') as c:
+        c.argument('key_type', options_list=['--regeneratekey'], required=True,
```

Comment:
For consistency with other CLI commands, change to just `--key`.

```
$ az storage account keys renew -h

Command
    az storage account keys renew: Regenerates one of the access keys for the specified storage
    account.

Arguments
    --account-name -n   [Required]: The storage account name.
    --key               [Required]: The key to regenerate.  Allowed values: primary, secondary.
```

Reply:
resolved